pkg/Microsoft.ML.AdditionalLearners/Microsoft.ML.AdditionalLearners.symbols.nupkgproj

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Pack">
+
+  <Import Project="Microsoft.ML.AdditionalLearners.nupkgproj" />
```

Comment:
>AdditionalLearners [](start = 32, length = 18)

MklDependentLearners maybe? it just separation of standard/additional learner is unclear from the names

Reply:
was thinking that we could use it for other learners, that we don;t want on the core package because of other dependencies. We could bundle them all in one package..  

---
In reply to: [206356744](https://github.com/dotnet/machinelearning/pull/611#discussion_r206356744) [](ancestors = 206356744)